src/index.html

```diff
@@ -6,13 +6,58 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product Card</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="../src/images/imac.jpeg"
+        alt="iMac 27 Retina"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
```

Comment:
There is no need to manually break title
It will automatically break into 2 lines
And with the correct font size and card width it will be done exactly the same as in test reference (and Figma)